include/opw_kinematics/opw_io.h

```diff
@@ -8,17 +8,27 @@ namespace opw_kinematics
 {
 
 template <typename T>
-std::ostream& operator<<(std::ostream& os, const Parameters<T>& params)
+std::ostream &operator<<(std::ostream &os, const Parameters<T> &params)
```

Comment:
Religious wars start over lesser transgressions!

Reply:
My apologies, my clang-format style was doing funky stuff and I forgot to fix it.